added functions bringToBack and bringToFront

granci opened the pull request:
added functions bringToBack and bringToFront to Overlay to work similarly as in Leaflet's ImageOverlay

granci committed:
added functions bringToBack and bringToFront

added functions bringToBack and bringToFront to Overlay to work similarly as in Leaflet's ImageOverlay

1tylermitchell commented:
Funny, I was already using this syntax and was wondering why it wasn't working - great to see your implementation, thanks for sharing. 
Does this also allow ordering of items in a layer control?

granci committed:
updated functions bringToBack and bringToFront

updated functions bringToBack and bringToFront for Overlay (added boolean 'isBack': false to options) and their further implementation also to Source and Layer (works just for 'tiled': false)

sheppard commented:
Thanks @granci, I rebased and merged these changes.
